app/views/reports/invoice_reports/index.html.erb

```diff
@@ -40,6 +40,9 @@
       <thead>
         <tr>
           <th class="text-right">Employee</th>
+          <% if @invoice_report.project.count > 1 %>
```

Comment:
Can we make this a helper method? `@invoice_report.project.count > 1` It's repeated several times here.